src/@next/components/organisms/ProductVariantPicker/types.ts

```diff
@@ -10,4 +9,6 @@ export interface IProps {
   ) => void;
   selectSidebar?: boolean;
   selectSidebarTarget?: HTMLElement | null;
+  queryAttributes: Record<string, string>;
+  updateUrlWithAttributes: (slug: string, value: string) => void;
```

Comment:
Name of this prop suggest that this component decides if url update should be performed. But from what I see it doesn't. If we change `updateUrlWithAttributes` to e.g. `onChangeAttribute`, the actual implementation on what to do when attribute with `slug` and `value` changes will be delegated to the higher order component, so component prop would be used in other scenarios, what now with `updateUrlWithAttributes` is logically possible, but semantically suggests the contrary. Am I right?

Reply:
you're right. i'll find better name for this function